vnext/ReactUWP/Views/TouchEventHandler.cpp

```diff
@@ -78,6 +78,13 @@ void TouchEventHandler::OnPointerPressed(
   if (!TagFromOriginalSource(args, &tag, &sourceElement))
     return;
 
+  // If this was caused by the user pressing the "back" hardware button, fire that event instead
+  if (args.GetCurrentPoint(sourceElement).Properties().PointerUpdateKind() ==
```

Comment:
See the other comment about listening for SystemNavigationManager.BackRequested. This feels a bit dirty right now.